### DISCUSSION TITLE
Thank you for your hard work

### DISCUSSION THREAD

**Discussion by 0xDEAD-C0DE** [YaLTeR/niri#188] [General]
I wanted to take this time to appreciate the effort @YaLTeR has invested in this project. I have admiration for you. For the way you have structured the organization of this project's code, for the great amount of reading you have done, for the time spent replying to us users, and for everything else. You appear as a very intelligent individual and a person that takes constructive feedback with a positive attitude. Thank you for everything!

**Comment by YaLTeR** [comment]
Thank you so much for the heartwarming words! :heart:

**Comment by ghost** [comment]
I literally just launched Niri for the first time (I used the Copr repository), after having switched from i3 to Gnome and missing tiling. Looks great! I've played with Rust a bit in the past but I want to get more serious now and help fix some bugs!

Thank you!
